### PR TITLE
Added bulk scan pay processor to AAT

### PR DESCRIPTION
**Pull request by nitinprabhu**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/BPS-806

### Change description ###

- Added bulk scan payment processor

- I have kept `enableKeyVaults` as `true` but let me know if that needs to be disabled as for now it is a repo with zero functionality.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: k8s/aat/common/rpe/bulk-scan-payment-processor.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: bulk-scan-payment-processor
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: bulk-scan-payment-processor
+    version: 1.0.0
```
Comment (adusumillipraveen) [inline k8s/aat/common/rpe/bulk-scan-payment-processor.yaml]:
```suggestion
    version: 0.0.2
```
this is the version of chart - https://github.com/hmcts/bulk-scan-payment-processor/blob/master/charts/bulk-scan-payment-processor/Chart.yaml#L6

File: k8s/aat/common/rpe/bulk-scan-payment-processor.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "f666440f-43c0-4abb-9c20-2ce175a374f0"
+      image: hmctspublic.azurecr.io/bulk-scan/payment-processor:prod-77831e9a
+      environment:
+        JAVA_TOOL_OPTIONS: -XX:InitialRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0
+          -XX:MinRAMPercentage=30.0 -XX:+UseConcMarkSweepGC -javaagent:/opt/app/applicationinsights-agent-2.5.0.jar
+      aadIdentityName: rpe
+      prometheus:
+        enabled: true
+    global:
```
Comment (adusumillipraveen) [inline k8s/aat/common/rpe/bulk-scan-payment-processor.yaml]: you don't need these 4 fields as you don't have keyvaults yet. Nothing wrong in having though.

File: k8s/aat/common/rpe/bulk-scan-payment-processor.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      readinessDelay: 45
+      readinessTimeout: 5
+      readinessPeriod: 15
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "f666440f-43c0-4abb-9c20-2ce175a374f0"
+      image: hmctspublic.azurecr.io/bulk-scan/payment-processor:prod-77831e9a
+      environment:
+        JAVA_TOOL_OPTIONS: -XX:InitialRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0
+          -XX:MinRAMPercentage=30.0 -XX:+UseConcMarkSweepGC -javaagent:/opt/app/applicationinsights-agent-2.5.0.jar
+      aadIdentityName: rpe
```
Comment (adusumillipraveen) [inline k8s/aat/common/rpe/bulk-scan-payment-processor.yaml]: Same here, nothing wrong in having though.

Comment (timja) [reply k8s/aat/common/rpe/bulk-scan-payment-processor.yaml]: it should probably go to defaults in the chart anyway though

File: k8s/aat/common/rpe/bulk-scan-payment-processor.yaml (hunk 2 above)
Comment (nitinprabhu) [reply k8s/aat/common/rpe/bulk-scan-payment-processor.yaml]: will keep those as we will have those soon

File: k8s/aat/common/rpe/bulk-scan-payment-processor.yaml (hunk 3 above)
Comment (nitinprabhu) [reply k8s/aat/common/rpe/bulk-scan-payment-processor.yaml]: will keep those as we will have those soon

**Pull request merged**